src/app/shared/wysiwyg/prompt.component.ts

```diff
@@ -18,15 +18,4 @@ import { Component } from '@angular/core';
   `,
   styleUrls: ['prompt.component.css']
 })
-export class PromptComponent {
```

Comment:
I made a terrible error in the way I managed visible state in this component. I was calling show and hide from the #prompt template variable in the parent component. This bypasses change detection. 

I could have passed visible in as an input, but I already had an ngIf in the parent, so I just used that. Seems like it makes no sense to have this as a separate component now, so I'm going to get rid of it.